packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -653,14 +658,15 @@ export default class BackupNg {
               schedule,
               logger,
               taskId,
+              settings,
```

Comment:
You can mutate `job.settings`.

Reply:
I'm not found of mutating objects when it can be avoided.